チャット画面実装

hidetoshi1230 opened the pull request:
#WHAT
チャット画面の実装

#WHY
サービスに必要な画面のため。

hidetoshi1230 committed:
チャット画面実装１（事前準備）

事前準備
・erb を haml へ変換
（gemファイル追記⇒インストール）
・ルーティングファイル追記（message#index）
・messagesコントローラー作成
・messagesコントローラー追記（index）
・index.html.hamlの作成
・application.scssの作成
・_variable.scssの作成
・_reset.scssの作成

hidetoshi1230 committed:
チャット画面実装2（レイアウト作成）

・index.html.hamlの編集
・messages.scssの編集

hidetoshi1230 committed:
チャット画面の実装（menu部分）

・index.html.hamlの編集
・messages.scssの編集
・Font AwesomeアイコンのGEMインストール
・application.scssの編集

hidetoshi1230 committed:
チャット画面実装３（チャットスペース部分）

・index.html.hamlの編集
・_messages.scssの編集
・_reset.scssの編集

TOMARUMARU commented on app/views/layouts/application.html.haml:
```diff
@@ -0,0 +1,10 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
```
ハッシュロケットは古い記法ですので修正しましょう！

TOMARUMARU commented on app/views/layouts/mailer.html.haml:
```diff
@@ -0,0 +1,8 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=utf-8", "http-equiv" => "Content-Type"}/
```
ハッシュロケットは古い記法ですので修正しましょう！

TOMARUMARU commented on config/routes.rb:
```diff
@@ -1,3 +1,3 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'chatspace' => 'messages#index'
```
ルーティングはresourcesで書くようにしましょう。

TOMARUMARU commented on app/assets/stylesheets/_messages.scss:
```diff
@@ -0,0 +1,169 @@
+html{
```
cssファイルを&記法を使って修正してください！

TOMARUMARU reviewed (commented):
view画面のスクリーンショットを貼ってください！

hidetoshi1230 committed:
チャット画面実装４（修正）

・cssファイルを&記法を使って修正してください！
・ハッシュロケットは古い記法ですので修正しましょう！
・ルーティングはresourcesで書くようにしましょう。

hidetoshi1230 commented:
![chatspace](https://user-images.githubusercontent.com/34529866/34317670-094b674e-e7f7-11e7-8361-5abff6c9e9ad.png)

TOMARUMARU reviewed (commented):
cssファイルですが、まだ&記法で修正できる場所が多々あるので修正してください。

hidetoshi1230 committed:
チャット画面実装５（修正）

cssファイルですが、まだ&記法で修正できる場所が多々あるので修正してください。

hidetoshi1230 committed:
チャット画面実装６（修正）

cssファイルですが、まだ&記法で修正できる場所が多々あるので修正してください。
